metaflow/plugins/aip/tests/flows/resources_flow.py

```diff
@@ -84,7 +84,7 @@ def get_env_vars(env_resources: Dict[str, str]) -> List[V1EnvVar]:
     )
 
 labels = {
-    "aip.zillowgroup.net/kfp-pod-default": "KF_POD_DEFAULT",
+    "aip.zillowgroup.net/aip-pod-default": "KF_POD_DEFAULT",
```

Comment:
I think I asked this on another PR but do we still need this label? cc @aaron-arellano

Reply:
this can be removed. I am not sure why the kfp-pod-default was being maintained. We use an argo specific label to target workflow pods in the webhook.